cmd/cloud/setup.go

```diff
@@ -407,6 +408,10 @@ func checkAPIToken(isDeploymentFile bool, platformCoreClient astroplatformcore.C
 	if len(claims.Permissions) == 0 {
 		return false, errNotAPIToken
 	}
+	if claims.ExpiresAt.Before(time.Now()) {
+		fmt.Printf("The given API Token %s has expired", claims.APITokenID)
```

Comment:
is there a better way to log the expired API token ID? Or do we even need to log the API token id?

Reply:
This should be fine. API Token ID would be helpful considering there could be multiple api tokens associated with a deployment